xapian-applications/omega/handler_libabw.cc

```diff
@@ -0,0 +1,168 @@
+/** @file handler_libabw.cc
+ * @brief Extract text and metadata using libarchive.
+ */
+/* Copyright (C) 2020 Parth Kapadia
```

Comment:
This is clearly significantly based on an existing file (e.g. all the `PARSE_FIELD` code is taken from one of the existing handler_*.cc files), so you should preserve copyright information from that file and add yourself rather than replacing it with yourself.

Reply:
Thank you for pointing this out. I have changed the copyright portion.